### PR TITLE
PEP 8 conventions

### PR DESCRIPTION
**Pull request by tusharmakkar08**
- Private functions named properly
- `sys.version` is used explicitly instead of  `ImportError` (`Explicit is better than implicit`) :grinning: 
- Line length made <= 120 char

### CODE REVIEW

**Comment by gleitz** [comment]
Thanks @tusharmakkar08! I'll review and come back with thoughts

File: howdoi/howdoi.py [hunk 1]
```diff
@@ -37,14 +27,18 @@
 from requests.exceptions import ConnectionError
 from requests.exceptions import SSLError
 
-# Handle unicode between Python 2 and 3
-# http://stackoverflow.com/a/6633040/305414
```
Comment (gleitz) [inline howdoi/howdoi.py]: Let's keep this stackoverflow link – it helps explain why we're defining the `u` function.

File: howdoi/howdoi.py [hunk 2]
```diff
@@ -18,16 +18,6 @@
...
-try:
-    from urllib.parse import quote as url_quote
-except ImportError:
-    from urllib import quote as url_quote
-
-try:
-    from urllib import getproxies
-except ImportError:
-    from urllib.request import getproxies
-
```
Comment (gleitz) [inline howdoi/howdoi.py]: Nice – I like what you did below. Much more explicit.

File: howdoi/howdoi.py [hunk 3]
```diff
@@ -59,8 +53,10 @@ def u(x):
 USER_AGENTS = ('Mozilla/5.0 (Macintosh; Intel Mac OS X 10.7; rv:11.0) Gecko/20100101 Firefox/11.0',
                'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:22.0) Gecko/20100 101 Firefox/22.0',
                'Mozilla/5.0 (Windows NT 6.1; rv:11.0) Gecko/20100101 Firefox/11.0',
-               'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_7_4) AppleWebKit/536.5 (KHTML, like Gecko) Chrome/19.0.1084.46 Safari/536.5',
-               'Mozilla/5.0 (Windows; Windows NT 6.1) AppleWebKit/536.5 (KHTML, like Gecko) Chrome/19.0.1084.46 Safari/536.5',)
+               'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_7_4) AppleWebKit/536.5 (KHTML, like Gecko) '
+               'Chrome/19.0.1084.46 Safari/536.5',
```
Comment (gleitz) [inline howdoi/howdoi.py]: It's a little hard to read the multiline string concatenation, especially if you miss the comma. How about [python line continuation](http://stackoverflow.com/a/1874679) using parenthesis instead?

Comment (tusharmakkar08) [inline howdoi/howdoi.py]: Parenthesis seems to be a nice solution. :+1:

File: howdoi/howdoi.py (hunk 1 above)
Comment (tusharmakkar08) [inline howdoi/howdoi.py]: Ok. I thought it was implicit while mentioning the `u` function. I will update the pull request correspondingly.

**Comment by tusharmakkar08** [comment]
I have updated the pull request with the necessary changes.

File: howdoi/howdoi.py (hunk 2 above)
Comment (tusharmakkar08) [inline howdoi/howdoi.py]: :+1:

**Comment by gleitz** [comment]
Put the stackoverflow link back in and I'll merge

**Comment by gleitz** [comment]
Oh nvm I see it. Thanks!

**Pull request merged**